test/accessibility/test_server_accessibility.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from pytest_axe.parameterize_tests import *  # NOQA
+
+
+class TestHomePageAccessibility(object):
+    params = {
```

Comment:
`params` are being accessed by the `pytest-axe` plugin. While it could be subject to change in the future, this is the current implementation that enables xfailing tests.

`pytest-axe` generates test functions at setup. The list of accessibility rules are fetched from aXe. This list is modified based on `params`, and used as input for `pytest_generate_tests`

https://github.com/mozilla-services/pytest-axe/blob/master/pytest_axe/parameterize_tests.py#L54-L62